cmd/infra/powervs/destroy.go

```diff
@@ -555,3 +619,68 @@ func deleteVpcSubnet(id string, v1 *vpcv1.VpcV1, infraID string) (err error) {
 	err = wait.PollImmediate(pollingInterval, vpcResourceDeletionTimeout, f)
```

Comment:
nit: this can just return wait.Poll.....

Reply:
modified.